index.js

```diff
@@ -95,94 +89,22 @@ module.exports = function(source) {
     }
   });
 
-  // TODO: This code could all still be perfectly valid with the new approach,
-  // look into it. -BB
-  /*
-  var shouldCacheImports = stylusOptions.importsCache !== false;
-
-  var importsCache;
-  if (stylusOptions.importsCache !== false) {
-    if (typeof stylusOptions.importsCache === 'object') {
-      importsCache = stylusOptions.importsCache;
-    } else {
-      if(!globalImportsCaches[configKey]) globalImportsCaches[configKey] = {};
-      importsCache = globalImportsCaches[configKey];
-    }
-  }
-
-  // Use input file system's readFile if available. The normal webpack input
-  // file system is cached with entries purged when they are detected to be
-  // changed on disk by the watcher.
-  var readFile;
-  try {
-    var inputFileSystem = this._compiler.inputFileSystem;
-    readFile = inputFileSystem.readFile.bind(inputFileSystem);
-  } catch (error) {
-    readFile = fs.readFile;
-  }
-  */
-
   // `styl.render`, promisified.
   var renderStylus = whenNodefn.lift(styl.render).bind(styl);
-  // webpack's `resolve`, promisified.
-  var resolve = whenNodefn.lift(self.resolve).bind(self);
 
   function tryRender() {
     return renderStylus().then(function(css) {
-      // TODO: Figure out how to add back sourcemap. -BB
-      /*
-      if (styl.sourcemap) {
-        styl.sourcemap.sourcesContent = styl.sourcemap.sources.map(function (file) {
-          return importPathCache.sources[path.resolve(file)]
-        });
-      }
-      */
```

Comment:
Decided to remove these commented out TODOs as I think we're okay, and can always refer to `stylus-loader` later.